src/backend/mod.rs

```diff
@@ -99,22 +99,13 @@ pub trait Keep {
 
 pub trait Thread {
     /// Enters the keep.
-    fn enter(&mut self) -> Result<Command<'_>>;
+    fn enter(&mut self, _gdblisten: &Option<String>) -> Result<Command>;
```

Comment:
Why the `_`?

Reply:
because of `unused arguments`, if feature gdb is not turned on

Reply:
Okay, so we at the minimum do not need that on the trait definition level